packages/core/types/src/modules/documents/params/document-engine.ts

```diff
@@ -69,7 +77,7 @@ export type Publish<TContentTypeUID extends UID.ContentType> = Pick<
 
 export type Unpublish<TContentTypeUID extends UID.ContentType> = Pick<
   TContentTypeUID,
-  'filters' | 'locale' | 'lookup'
+  'filters' | 'locale:string' | 'lookup'
```

Comment:
do we plan to support unpublishing many locales too? I find it limiting that you can publish many of them but not unpublish.

Reply:
Yep we do, we're just handling publishing first. Once everything is in place for
publishing it shouldn't be too much work to handle unpublishing